pkgs/top-level/all-packages.nix

```diff
@@ -13656,7 +13656,7 @@ in
 
   unixODBC = callPackage ../development/libraries/unixODBC { };
 
-  unixODBCDrivers = recurseIntoAttrs (callPackages ../development/libraries/unixODBCDrivers {});
+  unixODBCDrivers = recurseIntoAttrs (callPackages ../development/libraries/unixODBCDrivers { libmysql = mariadb-connector-c_2_3; });
```

Comment:
Why are you inconsistently referring to the package? Shouldn't it be `libmysql` everywhere?

Reply:
unixODBCDrivers.mariadb updated to 3.1.2